readthedocs/projects/models.py

```diff
@@ -1825,6 +1826,11 @@ class EnvironmentVariable(TimeStampedModel, models.Model):
         on_delete=models.CASCADE,
         help_text=_('Project where this variable will be used'),
     )
+    private = models.BooleanField(
```

Comment:
Hrm... In some places you are calling it `public` and in other `private` :smile: --that's why test are failing.

Reply:
hahaha whoops! It was `private` before, then /u/stsewd suggested calling it `public`!